updating links from fwlink to real link if exists (#17091)
dotnet/docs: docs/framework/wcf/diagnostics/tracing/trace-type-summary.md

ms.assetid: e639410b-d1d1-479c-b78e-a4701d4e4085
---
# Trace Type Summary

> [Source Levels](https://docs.microsoft.com/dotnet/api/system.diagnostics.sourcelevels&view=netframework-4.8) defines various trace levels: Critical, Error, Warning, Information, and Verbose, as well as provides description of the `ActivityTracing` flag, which toggles the output of trace boundary and activity transfer events.  

Reviewer: ```suggestion
[Source Levels](xref:System.Diagnostics.SourceLevels) defines various trace levels: Critical, Error, Warning, Information, and Verbose, as well as provides a description of the `ActivityTracing` flag, which toggles the output of trace boundary and activity transfer events.  
```

>  You can also review [TraceEventType](https://docs.microsoft.com/dotnet/api/system.diagnostics.traceeventtype&view=netframework-4.8) for the types of traces which can be emitted from <xref:System.Diagnostics>.  

Reviewer 2: Error here
```suggestion
 You can also review [TraceEventType](https://docs.microsoft.com/dotnet/api/system.diagnostics.traceeventtype?view=netframework-4.8) for the types of traces which can be emitted from <xref:System.Diagnostics>.  
```

>  You can also review [TraceEventType](https://docs.microsoft.com/dotnet/api/system.diagnostics.traceeventtype?view=netframework-4.8) for the types of traces which can be emitted from <xref:System.Diagnostics>.  

Reviewer: ```suggestion
 You can also review <xref:System.Diagnostics.TraceEventType> for the types of traces which can be emitted from <xref:System.Diagnostics>.  
```
